storage/maria/CMakeLists.txt

```diff
@@ -93,6 +99,11 @@ IF(WITH_UNIT_TESTS)
 
   ADD_SUBDIRECTORY(unittest)
 
+  IF(CMAKE_SYSTEM_NAME MATCHES AIX)
+    FOREACH(target ma_test1 ma_test2 ma_test3 ma_rt_test ma_sp_test test_ma_backup)
+      TARGET_LINK_OPTIONS(${target} PRIVATE "-Wl,-berok")
```

Comment:
Do you need these executables? What for? they are more or less dead weight, i not distributed and not tested. they just exist and are built without a particular reason.

Reply:
They are compiled if we activate unit tests. Without ``-berok``, build fails. Or we must compile without unittest.